lib/spack/spack/cmd/analyze.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+import llnl.util.tty as tty
+
+import spack.analyzers
+import spack.build_environment
+import spack.cmd
+import spack.cmd.common.arguments as arguments
+import spack.environment as ev
+import spack.fetch_strategy
+import spack.monitor
+import spack.paths
+import spack.report
+
+
+description = "analyze installed packages"
+section = "extensions"
+level = "short"
+
+
+def setup_parser(subparser):
+    subparser.add_argument(
+        '--overwrite', action='store_true',
+        help="re-analyze even if the output file already exists.")
+
+    monitor_group = spack.monitor.get_monitor_group(subparser)  # noqa
+
+    subparser.add_argument(
+        '-o', '--outdir', default=None,
+        dest='outdir',
+        help="write output to a different directory than ~/.spack/analyzers")
```

Comment:
`spack stage` has `-p` / `--path` -- should the UI be consistent with that?

Reply:
Yes I think so.